bootstrap/004_labels.sql

```diff
@@ -134,6 +134,59 @@ BEGIN
 
 END;
 
+CREATE OR REPLACE PROCEDURE INTERNAL.CREATE_PREDEFINED_LABEL(name text, grp text, rank number, condition text)
```

Comment:
I dont think we need this for the predefined labels. We should just be able to insert direclty in the other proc.

Reply:
Earlier one, I used direct "insert into" to populate the predefined labels.  I realized that using a stored proc could apply the same checks over "name", "condition" or other properties in the same way as the regular labels. That's the main reason I choose to use a stored procedure to 1) check the input, 2) insert into table.